cmplibrary/src/main/java/com/sourcepoint/cmplibrary/model/exposed/SpConfig.kt

```diff
@@ -50,6 +53,13 @@ data class SpCampaign(
 
 data class TargetingParam(val key: String, val value: String)
 
+fun List<TargetingParam>.toJsonElement() = JsonConverter.converter.encodeToJsonElement(
+    fold(mutableMapOf<String, String>()) { acc, targetingParam ->
```

Comment:
`acc` is a bit confusing naming

Reply:
Yeah, I can rename it. But this is a common name when implementing a reduce (fold) function. Acc stands for accumulator, which is the collection that you're reducing into.